packages/messages/_common.scss

```diff
@@ -0,0 +1,47 @@
+$types: info $info $info--darkbg, warning $advarsel $advarsel--darkbg, error $feil $feil--darkbg,
+    success $suksess $suksess--darkbg;
+
+@mixin generate-message-colors($component) {
```

Comment:
Vet ikke om jeg er så fan av dette 😕 Er for så vidt en elegant løsning, men jeg tror den abstraherer vekk litt for mye av hva som skjer: Man må tolke relativt komplisert Sass-syntaks for å forstå hvilke farger som brukes hvor, og fargeverdiene blir aldri knyttet til en funksjon slik de gjør som vanlige Sass-variabler.

Reply:
Ja, enig. Jeg går tilbake til en litt mindre overkill approach 👍

Reply:
Forenklet den nå, og det ble mye mer lesbart. Mindre fancy tho :(